aify/index.html

```diff
@@ -129,24 +145,167 @@ <h3 class="feature-title mt-0">Discover</h3>
+                                            <h5 class="feature-title mt-0">Medium scale business owners</h5>
+                                            <!-- <p class="text-sm mb-0">Lorem ipsum dolor sit amet, consecte adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua dui.</p> -->
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="col-lg-4">
+                                    <div class="row zoom">
+                                        <img src="images/enterprise.png" alt="" width="32%" id="like">
+                                    </div>
+                                    <br><br>
+                                    <div class="row">
+                                        <div class="feature-content">
+                                            <h5 class="feature-title mt-0">Enterprise and Industries</h5>
+                                            <!-- <p class="text-sm mb-0">Lorem ipsum dolor sit amet, consecte adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua dui.</p> -->
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="cta section" id="integrate">
                 <div class="container-sm">
                     <div class="cta-inner section-inner">
                         <div class="cta-header text-center">
-                            <h2 class="section-title mt-0">Get it and Switch</h2>
-                            <p class="section-paragraph">Lorem ipsum is common placeholder text used to demonstrate the graphic elements of a document or visual presentation.</p>
+                            <h2 class="section-title mt-0">Integrate it</h2>
+                            <br>
+                            <div class="feature is-revealing">
+                                <div class="feature-inner zoom2">
+                                    <!-- <img src="images/AIFY.png" alt=""> -->
+                                    <img src="images/carbon.png" alt="">
+
+                                </div>
+                            </div>
+
+                            <br>
+                            <p class="section-paragraph">Can be integrated with any programming language and can be used both as an API and a tool even on websites or apps hosted on static platforms.
+                            </p>
                             <div class="cta-cta">
-                                <a class="button button-primary" href="#">Buy it now</a>
+                                <!-- <a class="button button-primary" href="#">Buy it now</a> -->
```

Comment:
Can we keep this button and send users to the documentation at docs.thecodefoundation.dev?